earth_enterprise/src/fusion/autoingest/AssetVersion.h

```diff
@@ -137,6 +137,8 @@ class AssetVersionImpl : public AssetVersionStorage, public StorageManaged {
     return (state == AssetDefs::Bad);
   }
 
+  virtual bool InputStatesAffectMyState(AssetDefs::State stateByInputs, bool blockedByOfflineInputs) const {assert(false); return true;}
+  virtual bool ChildStatesAffectMyState() const {assert(false); return true;}
```

Comment:
Minor: you could get rid of the `assert(false)` here and then you wouldn't have to define the function in LeafAssetVersionImplD.

Reply:
Removed `ChildStatesAffectMyState`.